crates/atomic/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Dawid Ciężarkiewicz <dpc@dpc.pw>"]
 description = "Atomic run-time controllable drain for slog-rs"
 keywords = ["slog", "logging", "log",  "atomic"]
 license = "MPL-2.0"
-documentation = "https://dpc.github.io/slog-rs/"
+documentation = "https://docs.rs/slog-atomic"
```

Comment:
You must decide which links to use.

Long story about this PR: In my first steps with rust lang I'm much more read documentation and try examples from it rather then works with source code of foreign crates. I choose `slog` for logging in my app. And I want to access log from any point of my code. I discover `slog-scope`, but on crates.io `slog-scope` documentation link is broken. When googling `slog-scope` found link to https://docs.rs/crate/slog-scope/  And on the index page of the docs for `slog-scope` presented example when and how to use it! that's what I need!

In my case for all sub-crates in repo of `slog` links must be like that:
 - home `https://github.com/slog-rs/slog`
 - documentation `https://docs.rs/slog-<create>`

Ideal story for sub-crate documentation when it with examples how to use it.
Specially for beginners like me )

Sorry for my poor English.